server/controller/root.py

```diff
@@ -106,6 +110,63 @@ def historical():
     }
 
 
+@get('/upload_logs')
+@kajiki_view('upload_logs')
+@authenticated()
+def upload_logs():
+    api_key = config.user_management_interface.get_api_key_for_user(request)
```

Comment:
@amandagiles95 , get rid of this line, it is automatically added by the `@authenticated` decorator.